src/Domain/ApplicationTime.php

```diff
@@ -69,6 +70,10 @@ public static function setTime(int $appTime = null)
     {
         static::$appTime = DateTimeImmutable::createFromFormat('U', $appTime ?? time())
             ->setTimezone(new DateTimeZone('UTC'));
+
+        if (class_exists(Chronos::class)) {
```

Comment:
`setTime` accepts a timestamp. I'm not sure how you would use it to set _just_ a date. I can also set the Chronos `Date` if you like.